gradle/shipkit.gradle

```diff
@@ -0,0 +1,40 @@
+shipkit {
+    git.releasableBranchRegex = "release/.+"
+    git.user = "jaeger-release-bot"
+    git.email = "jaeger-client-java@jaegertracing.io"
+
+    gitHub.repository = "jaegertracing/jaeger-client-java"
+    gitHub.readOnlyAuthToken = System.getenv("GH_READ_TOKEN") ?: "7daade13ce3b15302a0da78abcb57ba2d8593d12"
+    gitHub.writeAuthToken = System.getenv("GH_WRITE_TOKEN")
+    releaseNotes.file = "CHANGELOG.md"
+
+    // see https://github.com/jaegertracing/jaeger-client-java/settings/collaboration and
+    // https://github.com/orgs/jaegertracing/teams/jaeger-maintainers/members
+    // sorted alphabetically by username
+    team.developers = [
+            'black-adder:Won Jun Jang',
```

Comment:
Where is this being used? 

Could we just point it to https://gitter.im/jaegertracing/Lobby?

Reply:
No, it's required by Maven Central.

Reply:
There is no need to include everybody. Some OSS projects use link to forum or chat to avoid including all people. I guess we could use `gitter: https://gitter.im/jaegertracing/Lobby`

Reply:
https://maven.apache.org/pom.html#Developers

Reply:
https://github.com/opentracing/opentracing-java/blob/master/pom.xml#L94

Reply:
I'd rather not have this there, but if it's required, then let's please have the appropriate data there, not just some data to please Central...

Reply:
i don't mind it was just a suggestion. It also makes it simpler to get in touch

Reply:
A downside of listing all the core developers there, is that this has the potential to get outdated easily. Let's try to get the right thing first and, if it does get outdated often, we can consider adding some random data (like the gitter channel).